requirements.txt

```diff
@@ -2,3 +2,4 @@ featuretools>=0.5.1
 scikit-learn>=0.20.1
 bokeh>=1.0.2
 jupyter>=1.0.0
+pandas==0.23.4
```

Comment:
We pinned pandas to 0.23.4 in the latest release of featuretools, no need to add it to the requirements